composer.json

```diff
@@ -1,31 +1,25 @@
 {
-  "name":        "tijsverkoyen/css-to-inline-styles",
-  "type":        "library",
+  "name": "tijsverkoyen/css-to-inline-styles",
+  "type": "library",
   "description": "CssToInlineStyles is a class that enables you to convert HTML-pages/files into HTML-pages/files with inline styles. This is very useful when you're sending emails.",
-  "homepage":    "https://github.com/tijsverkoyen/CssToInlineStyles",
-  "license":     "BSD",
-  "authors":     [
+  "homepage": "https://github.com/tijsverkoyen/CssToInlineStyles",
+  "license": "BSD-3-Clause",
+  "authors": [
     {
-      "name":  "Tijs Verkoyen",
+      "name": "Tijs Verkoyen",
       "email": "css_to_inline_styles@verkoyen.eu",
-      "role":  "Developer"
+      "role": "Developer"
     }
   ],
-  "require":     {
-    "php":                  ">=5.3.0",
-    "symfony/css-selector": "~2.1"
+  "require": {
+    "symfony/css-selector": "^2.7"
   },
   "require-dev": {
-    "phpunit/phpunit":      "~4.0"
+    "phpunit/phpunit": "~4.8"
```

Comment:
you should also allow phpunit 5, not only 4.8 (4.8 does not officially support PHP 7, even if it mostly works)